UnityProject/Assets/Scripts/HealthV2/Living/Mutations/Metabolism/ToxicResistance.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using HealthV2;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "ToxicResistance", menuName = "ScriptableObjects/Mutations/ToxicResistance")]
+public class ToxicResistance : MutationSO
+{
+	public override Mutation GetMutation(BodyPart BodyPart,MutationSO _RelatedMutationSO)
+	{
+		return new InToxicResistance(BodyPart,_RelatedMutationSO);
+	}
+
+	private class InToxicResistance: Mutation
+	{
+
+		public BodyPart Related;
+
+		public InToxicResistance(BodyPart BodyPart,MutationSO _RelatedMutationSO) : base(BodyPart,_RelatedMutationSO)
+		{
+
+		}
+
+		public override void SetUp()
+		{
+			Related = BodyPart;
+			Related.damageWeaknesses.Tox += -0.66f; // so if 1 == 0.33%
```

Comment:
it would be better to expose that number  to editor  as a serialised field, so we can easily tweak them and create variations.

Reply:
I'm not 100% sure if the system would handle multiple SOs of the same type

Reply:
yeah, should work with multiple of the same so